adivinar_numero.rb

```diff
@@ -0,0 +1,11 @@
+n = rand(1..10)
+puts "Bienvenido al programa, Adivina el número que estoy pensando"
```

Comment:
ruby convention is `'` for string without interpolation and `"` for interpolations